mdata/datadict.md

```diff
@@ -169,36 +169,36 @@ the creation phase of zero-touch custom image management.
 
 ## Recommendations For Third Party Vendors
 
-If you are producing an image to be deployed in a Triton instance, or
-specifically in the Joyent Public Cloud, you should follow the rules outlined
-previously in this document with respect to key names and intended behaviour.
-In addition, if you wish to provide additional configuration options that are
-specific to your product, you should do so with your own key namespace prefix.
+If you are producing an image to be deployed in a Triton instance, you should
+follow the rules outlined previously in this document with respect to key names
+and intended behaviour. In addition, if you wish to provide additional
+configuration options that are specific to your product, you should do so with
+your own key namespace prefix.
 
 To ensure that there are not inter-vendor key name conflicts, vendors are
 encouraged to use the reversed version of their corporate DNS domain as a
 prefix.  For example:
 
     com.ubuntu:user-data
-    
+
         or
-    
+
     com.riverbed:stingray-license-key
 
 <!-- Links/References -->
 
 [1]: http://www.smartos.org/
 
-[2]: https://github.com/joyent/mdata-client
+[2]: https://github.com/tritondatacenter/mdata-client
 
 [3]: https://en.wikipedia.org/wiki/Shebang_%28Unix%29
 
-[4]: https://www.joyent.com/triton/compute
+[4]: https://www.tritondatacenter.com/triton/compute
 
 [5]: https://en.wikipedia.org/wiki/Universally_unique_identifier
 
 [6]: protocol.html
 
-[7]: https://github.com/joyent/triton-cns
+[7]: https://github.com/tritondatacenter/triton-cns
 
-[8]: https://smartos.org/man/1M/mdata-list
+[8]: https://smartos.org/man/8/mdata-list
```

Comment:
This link returned not-found.

Reply:
Yeah, but it’s the new canonical place. I’ll be updating the target so that the link will work.

Reply:
Ah ok. Then all is good!